exporters/otlp/CMakeLists.txt

```diff
@@ -33,7 +33,14 @@ if(BUILD_TESTING)
     TARGET recordable_test
     TEST_PREFIX exporter.otlp.
     TEST_LIST recordable_test)
-  find_library(GMOCK_LIB gmock PATH_SUFFIXES lib)
+  if(MSVC)
+    add_definitions(-DGTEST_LINKED_AS_SHARED_LIBRARY=1)
+  endif()
+  if(MSVC AND CMAKE_BUILD_TYPE STREQUAL "Debug")
+    find_library(GMOCK_LIB gmockd PATH_SUFFIXES lib)
+  else()
+    find_library(GMOCK_LIB gmock PATH_SUFFIXES lib)
+  endif()
   if(GMOCK_LIB)
```

Comment:
I also think that this check `if(GMOCK_LIB)` - probably needs to be removed entirely. If we are building with tests enabled, we should `require` that GMock is available. If it's not there, we should fail building `otlp_exporter_test` instead of silently skipping it.